datasets/creating-dataset.mdx

```diff
@@ -0,0 +1,198 @@
+    - `Description` is an optional brief description of the field. You can use it to provide more context and details about the data.
+    - `Example value` is an optional example for this field. It can be useful for documentation purposes.
+
+    <Frame>
+      <img src="/assets/console/datasets-create-temporal-light.png" alt="Tilebox Console" className="dark:hidden" />
+      <img src="/assets/console/datasets-create-temporal-dark.png" alt="Tilebox Console" className="hidden dark:block" />
+    </Frame>
+  </Step>
+  <Step title="Create the dataset">
+    Once you're done completing the fields, click "Create" to create the dataset. You are redirected to your newly created dataset.
+  </Step>
+</Steps>
+
+The next steps are to create at least one collection and ingest data into it.
+// TODO write docs for that
+
+It might then resemble this:
+<Frame>
+  <img src="/assets/console/datasets-overview-light.png" alt="Tilebox Console" className="dark:hidden" />
+  <img src="/assets/console/datasets-overview-dark.png" alt="Tilebox Console" className="hidden dark:block" />
+</Frame>
+
+## Available types
+
+Tilebox supports primitives and data for data fields. These are defined in [well_known_types.proto](https://github.com/tilebox/tilebox-go/blob/main/apis/datasets/v1/well_known_types.proto).
+Tilebox also uses Google well-known-types [Duration](https://protobuf.dev/reference/protobuf/google.protobuf/#duration) and [Timestamp](https://protobuf.dev/reference/protobuf/google.protobuf/#timestamp).
+
+<Note>
+  If there is a data type you want to see in Tilebox,
+  <a href="mailto:support@tilebox.com">please get in touch</a>.
+</Note>
+
+Here's how to import the types.
+
+<Tabs>
+<Tab title="Python">
+```python Python
+from google.protobuf import duration_pb2, timestamp_pb2
+from tilebox.datasets.datasetsv1 import well_known_types_pb2
+```
+</Tab>
+
+<Tab title="Go">
+```go Go
+import (
+    datasetsv1 "github.com/tilebox/tilebox-go/protogen/go/datasets/v1"
+    "google.golang.org/protobuf/types/known/durationpb"
+    "google.golang.org/protobuf/types/known/timestamppb"
+)
+```
+</Tab>
+</Tabs>
+
+### Table of types per language
+
+| Tilebox Type   | Python Type | Go Type     |
+| -------------- | ----------- | ----------- |
+| string         | str /  object? | string      |
+| int64          | int         | int64       |
+| uint64         | np.uint64   | uint64      |
+| float64        | np.float32  | float64     |
+| bool           | bool        | bool        |
+| bytes          | ?       | []byte      |
+| Duration       |             |     |
+| Timestamp      |             |     |
+| UUID           |             |     |
+| LatLon         |             |     |
+| LatLonAlt      |             |     |
+| GeobufData     |             |     |
+
+### Primitive types
+
+`string` represents a string of characters.
+
+example : "Some string"
```

Comment:
Can you use the same component we use here: 
http://localhost:3000/datasets/timeseries#common-fields

I think the formatting of that looks much better than just a plain prose listing.

Reply:
just checked, its `ParamField`

<ParamField path="string" type="Some string">
  represents a string of characters.
</ParamField>